packages/core/src/stream-loading/__tests__/stream-loader.test.ts

```diff
@@ -65,8 +65,9 @@ describe('StreamLoader querying against real Ceramic node', () => {
 
     dispatcherIpfs = await createIPFS()
     dispatcher = await createDispatcher(dispatcherIpfs, TOPIC)
-    // speed up how quickly the dispatcher gives up on loading a non-existent commit from ipfs.
-    dispatcher._ipfsTimeout = 100
+    // We need this timeout to be short enough that the non-existent lookup timesout before
+    // the test case itself timesout.
+    dispatcher._ipfsTimeout = 1000
```

Comment:
why increasing this timeout?

Reply:
A 100ms timeout was too short for the rest of the test cases. This timeout applies to about 7 tests that all share the dispatcher. It wasn't long enough to fetch blocks over bitswap.

Reply:
oh, that's surprising.  100ms feels like a long time for well-connected nodes.  This also has never been a problem with kubo.  Is rust bitswap a lot slower?

Reply:
Honestly I don't concretely know why rust is sensitive to 100ms timeouts, but I do have a few theories. However my perspective is these are correctness tests not performance tests and so increasing the timeout to get stable tests was best. 

I did however test one of my theories that its TCP thats slow compared to QUIC. I changed rust-ceramic to use QUIC exclusively for these tests and it passes consistently with 200ms timeout. At 100ms it still had ~5% failure rate. I can make the QUIC change and use 200ms thoughts?